### ISSUE TITLE
404 for "/footprint"

### ISSUE THREAD

**Issue by johnwjlim** [plastic-oceans461/plastic-oceans461.github.io#28]
![image](https://user-images.githubusercontent.com/34074206/40901916-f7db24d0-6786-11e8-938f-219c88f3315c.png)

Reproduction steps
1) Go to discover page
2) click on footprint

**Comment by lisakoss** [comment]
Can you provide more information on how you received this error? It's not showing up on my end.

**Comment by lisakoss** [comment]
Awesome, good to know! Thanks for double checking for us! :)

**Comment by lisakoss** [comment]
Actually we figured out the root cause of this error. Our nginx build pack that heroku depends on was searching for a default index.html page to render, but since we're giving this page to the users via the client side rather than the server side, users who had never visited the main URL of our site could not visit any other pages on our site without experiencing a 404 served by nginx (for ex. if users visited https://plastic-oceans.herokuapp.com/discover before visiting https://plastic-oceans.herokuapp.com/ they would receive the 404 error). 

We have added a static.json file into the root directory of our project that fixes this issue and tells nginx where to look for the index if you're trying to visit one of these pages now. Now every user should be redirected to our home page and should no longer ever see this 404 error. :)
